torch/functional.py

```diff
@@ -1262,7 +1262,12 @@ def norm(input, p="fro", dim=None, keepdim=False, out=None, dtype=None):  # noqa
         more consistent with NumPy's numpy.linalg.norm.
 
     Args:
-        input (Tensor): the input tensor
+        input (Tensor): The input tensor. The data type must be either floating
+            point or complex. For complex inputs, the norm is calculated on the
+            absolute values of each element. If the input is complex and neither
+            :attr:`dtype` nor :attr:`out` is specified, the result's data type will
+            be the corresponding downgraded real number type.
```

Comment:
"corresponding downgraded real number type" -> "corresponding floating point type (e.g. float if :attr:\`input\` is complexfloat)."